src/main.js

```diff
@@ -1,8 +1,10 @@
 import Vue from 'vue'
 import App from './App.vue'
+import store from './store';
 
 Vue.config.productionTip = false
 
 new Vue({
   render: h => h(App),
+  store,
```

Comment:
notice this import, that is how you will have access to store and be able to do map...

Reply:
Ahh this is so helpful, I knew I missing something!